### ISSUE TITLE
OSM editing plugin - Save changes uploaded to OSM on the local map

### ISSUE THREAD

**Issue by Almorca** [osmandapp/OsmAnd#13398]
Currently when an edit is made and uploaded to OSM the edit is lost on the local map.

It would be an improvement if the change made through the plugin would also apply to the downloaded map.

**Comment by vshcherb** [comment]
Yes, it would be great, though OsmAnd can't generate OBF maps itself, so it requires server interaction as now. So it's quite tricky to fix it.

**Comment by KronosTheLate** [comment]
Big +1 here. Usually i add places I just visited and want to bookmark for my personal collection. Withthe addedPOI dissapearing, I dont really have good options for bookmarking that POI.
